.github/workflows/ci.yml

```diff
@@ -1,9 +1,13 @@
 name: CI
 
 on:
+  schedule:
```

Comment:
Why do we need this schedule?

Reply:
For nightly builds to detect flaky tests or issues caused by automaticly updated components.